release/src/router/www/state.js

```diff
@@ -1423,6 +1423,15 @@ function showMenuTree(menuList, menuExclude){
 						menu_code += firstEntry;
 						menu_code += ', this);" title="';
 						menu_code += curMenu.tab[firstEntry].url;
+						if(isSwMode("re")){
```

Comment:
from where are these coming line coming from?

Reply:
they are coming from my hands, I did search inside the code for something that could make the wizard working as you want, I ended up that something must tell the wizard page the current operating mode to work correctly, nothing inside the code seems to do that so I written the missing lines

Reply:
awesome!